### PR TITLE
Update fr-projects.json

### PR DESCRIPTION
**Pull request by Fend4self**
adblock-edge + adium  FR translaction correction+addition

### CODE REVIEW

**Comment by vyp** [comment]
Pretty sure the translations to "web browser addons", "experimental" and "instant messaging" are not needed because they are already done in [source/locales/fr.json](https://github.com/nylira/prism-break/blob/master/source/locales/fr.json), but I may be wrong.

**Comment by nylira** [comment]
What @vyp said is right. Please edit out those changes, and then I'll sync this.

**Comment by Fend4self** [comment]
Sorry, it wasn't clear to me wether we should edit from ./source/db/*-projects.json or ./source/locales/*.json

File: source/db/fr-projects.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
       {
         "name": "OS X",
         "subcategories": [
-          "Instant Messaging",
+          "instant messaging",
```
Comment (nylira) [inline source/db/fr-projects.json]: Capitalize this to revert it to the previous version.

File: source/db/fr-projects.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
       {
         "name": "Android",
         "subcategories": [
-          "Web Browser Addons"
+          "web browser addons"
```
Comment (nylira) [inline source/db/fr-projects.json]: And this one too!

**Comment by Fend4self** [comment]
Done

**Pull request merged**